tests/evaluation/evaluation_formula_array.test.ts

```diff
@@ -884,4 +886,39 @@ describe("evaluate formulas that return an array", () => {
       expect(model.getters.getSpreadZone({ sheetId, col: 0, row: 0 })).toEqual(toZone("A1:B2"));
     });
   });
+
+  describe("result array can collides with merged cells", () => {
+    test("throw error message when encountering a vertically merged cell", () => {
+      setCellContent(model, "A1", "=MFILL(2, 2, 42)");
```

Comment:
not sure how this test and the one below are relevant. just testing the rpesence of a merge within the spreading range should be enough